packages/http/src/models/Hotspot.ts

```diff
@@ -115,7 +121,12 @@ export default class Hotspot extends DataModel {
     this.lng = hotspot.lng
     this.lat = hotspot.lat
     this.block = hotspot.block
-    this.status = hotspot.status
+    this.status = {
+      gps: hotspot.status?.gps || '',
+      height: hotspot.status?.height || 0,
```

Comment:
Does the API return 0 or null? Might be good to tell the difference between one that is undefined and one that is actually 0?

Reply:
I'm not certain, but I think in this scenario null and 0 have the same meaning (sync hasn't started)

Reply:
Yeah I think it should be ok